bodhi/server/models/models.py

```diff
@@ -1247,7 +1247,13 @@ def send_update_notice(self):
             mailinglist = config.get('%s_announce_list' % release_name)
         elif self.status is UpdateStatus.testing:
             mailinglist = config.get('%s_test_announce_list' % release_name)
-        templatetype = '%s_errata_template' % release_name
+        
+        # switch email template to legacy if update aims to EPEL <= 7
+        if release_name == 'fedora_epel' and self.release.version <= 7:
+            templatetype = '%s_errata_legacy_template' % release_name
+        else:
+            templatetype = '%s_errata_template' % release_name
```

Comment:
It does seem that we lack tests on this method, but we do have a test for the class this method is part of here:

https://github.com/fedora-infra/bodhi/blob/dd6add3cfa1392135eb972416f2b892152147850/bodhi/tests/server/models/test_models.py#L172

Could you add unit tests that hit these two paths on the ```send_update_notice()``` method? You don't have to worry about the rest of the method unless you just want to get some testing done while you are in there, but getting some coverage at least on this new if statement would be good.